sdk/storage/azure-storage-blob/swagger/README.md

```diff
@@ -617,4 +617,64 @@ directive:
     $.description = "The Put Blob from URL operation creates a new Block Blob where the contents of the blob are read from a given URL.  This API is supported beginning with the 2020-04-08 version. Partial updates are not supported with Put Blob from URL; the content of an existing blob is overwritten with the content of the new blob.  To perform partial updates to a block blob's contents using a source URL, use the Put Block from URL API in conjunction with Put Block List.";
 ```
 
+### Rename ListBlobsIncludeItem Enums to be underscore cased
+```yaml
+directive:
+  - from: swagger-document
+    where: $.parameters.ListBlobsInclude
+    transform: >
+      $.items["x-ms-enum"].values = [
+        {
+          "value": "copy",
+          "name": "copy",
+          "description": ""
```

Comment:
Long-term descriptions can be added here to provide better Javadocs than the default `/** Enum value <value>. */`